<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstTimeWizard</class>
 <widget class="QWizard" name="FirstTimeWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>First-time configuration</string>
   </property>
   <property name="subTitle">
    <string>The Arduino IDE needs some information in order to work correctly.&lt;br /&gt;Please fill them in the form below:</string>
   </property>
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>I need the location of your Arduino installation.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Arduino path:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="arduinoPathEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="arduinoPathButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;img src=&quot;:/images/16x16/help-hint.png&quot; /&gt;If you don't have one, you can download the software for free on &lt;a href=&quot;http://arduino.cc/&quot;&gt;the Arduino website&lt;/a&gt;.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>&lt;hr /&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>The sketchbook folder is where the IDE will look for existing projects.&lt;br /&gt;It will be created if it does not already exist.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Sketchbook path:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="sketchbookPathEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="sketchbookPathButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>Configuration successful</string>
   </property>
   <property name="subTitle">
    <string>The Arduino IDE is now ready.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>The configuration is complete, you may now start using the IDE.&lt;br /&gt;Thank you for using the software. Enjoy!</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>The Arduino IDE is an unofficial project by Denis Martinez.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>More information at:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="urlLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
